README.md

```diff
@@ -0,0 +1,18 @@
+# DANDI Handbook
+Handbook for interacting with the DANDI archive.
+
+## DANDI Style Guidelines
+
+Follow the guidelines below when creating and revising text in the DANDI Handbook:
+
+*	**dandi-** repositories (hyphenate, dandi-archive, dandi-cli, *not* dandischema; Dandisets is an exception because that is a
```

Comment:
```suggestion
*	The names of most DANDI GitHub repositories are hyphenated (dandi-archive, dandi-cli; does not apply to dandischema); "Dandisets" is an exception because that is a
```

Reply:
@jwodder, I think the idea here was to standardize even `dandischema`. Is that one excepted from the hyphenation pattern because it is associated with a Python package?

Reply:
I also noticed I had an extra comma in here, I will rewrite to clarify, thanks.

Reply:
> It's excepted from hyphenation because the name of the repository isn't hyphenated. Either way, the item needs to be rewritten to be clearer about dandischema.

I am sure you realize the circularity of your statement. Repository names can be changed to match up with a style guide, or style guides can be changed to except specific repository names--but both of those plans would require some justification.

My question is whether the repository name *needs* to remain unhyphenated for some other reason (such as wanting to keep it the same as the resulting Python package). If not, I'm proposing that we follow the style guide and change the name of the repo.

Reply:
@jwodder & @waxlamp - I am thinking I should revise as follow:

```suggestion
*	DANDI repository names - follow the spelling that is used in GitHub (dandi-archive and dandi-cli are hyphenated, dandischema and dandisets are not hyphenated)
```

Reply:
> @waxlamp - i don't think anything prevents us from hyphenating dandischema. the library name can be decoupled from the repo name. there may be some files we may need to modify if some test or dockerfile directly references the reponame. github, i believe, offers redirects from old names for a while.

@waslamp - if @satra is ok with changing the repo names to match the convention (i.e. changing "dandischema" to  "dandi-schema") then I will revise to say 
```suggestion
*	**dandi-** repository names should be hyphenated (e.g. dandi-archive, dandi-cli, dandischema); Dandisets is an exception because that is a
```

Reply:
Let's go ahead and change it.

@melster1010, in your suggested change, did you mean to change `dandischema` to `dandi-schema`?